js/task-controller.js

```diff
@@ -0,0 +1,104 @@
+export default class TaskController {
+
+    _tasks = [];
+
+    currentTasks_container = document.querySelector("#currentTasks");
+    completedTasks_container = document.querySelector("#completedTasks");
+
+    setTasks(newTasks) {
+        this._tasks = newTasks;
+    }
+
+    getTasks() {
+        return this._tasks;
+    }
+
+    addTask(task) {
+        const newTasks = this.getTasks();
+        newTasks.push(task);
+        this.setTasks(newTasks);
+    }
+
+    updateTask(props) {
+        const currentTasks = this.getTasks();
+
+        const { index, title, text, priority } = props;
+        currentTasks[index].title = title;
+        currentTasks[index].text = text;
+        currentTasks[index].priority = priority;
```

Comment:
it's simpler and more clear with `map` method

Reply:
Thanks, fixed
```suggestion
          const newTasks = currentTasks.map(task => {
            if (task.id == id) {
                task.title = title;
                task.text = text;
                task.priority = priority;
            }
            return task;
        });

```